client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
@guerler Don't worry about tours at all -- multipage/reloads should actually work fine w/ bootstrap tours.